app/recordtransfer/static/recordtransfer/js/submission_form/index.js

```diff
@@ -15,25 +15,61 @@ import {
     setupHelpTooltips,
 } from "./widgets";
 
-document.addEventListener("DOMContentLoaded", function () {
+const _setupWithContext = () => {
+    const contextElement = document.querySelector("[id^=\"js_context_\"]");
+    if (!contextElement) {
+        return;
+    }
+    const context = JSON.parse(contextElement.textContent);
+    
+    const idParts = contextElement.id.split("js_context_");
+    const contextFor = idParts.length > 1 ? idParts[1] : "";
```

Comment:
This is a little bit hard to read. How about this?

```suggestion
    const contextFor = contextElement.id.replace(/^js_context_/, "");
```

Reply:
That looks better. I've changed that here: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/550/commits/0d3e8cd7643c730276fdd2cc6985bf8610c366eb